modules/serverless-install-web-console.adoc

```diff
@@ -0,0 +1,50 @@
+// Module included in the following assemblies:
+//
+// * /serverless/installing_serverless/installing-openshift-serverless.adoc
+
+[id="serverless-install-web-console_{context}"]
+= Installing the {ServerlessOperatorName}
+
+This procedure describes how to install and subscribe to the {ServerlessOperatorName} from the OperatorHub using the {product-title} web console.
+
+.Procedure
+
+. Navigate in the web console to the *Catalog* → *OperatorHub* page.
```

Comment:
Reword this to keep consistency with a similar step above:
In the {product-title} web console, navigate to the *Catalog* → *OperatorHub* page.
https://github.com/openshift/openshift-docs/pull/21074/files#diff-aa00c6cbca126bf4d6ddf98f09f49e5aR25